custom/icds_reports/static/js/directives/download/download.directive.js

```diff
@@ -470,7 +503,8 @@ function DownloadController($rootScope, $location, locationHierarchy, locationsS
 
     vm.isVisible = function(level) {
         return level === 0 || (vm.selectedLocations[level - 1] && vm.selectedLocations[level - 1] !== 'all') &&
-            !(vm.isIncentiveReportSelected() && level > 2) && !(vm.isLadySupervisorSelected() && level > 2);
+            !(vm.isIncentiveReportSelected() && level > 2) && !(vm.isLadySupervisorSelected() && level > 2) &&
+            !(vm.isTakeHomeRationReportSelected() && level > 3);
```

Comment:
This is only available at sector and above?

Reply:
Yes, only awc is excluded.